docker/lr-gcloud-samtools/Makefile

```diff
@@ -0,0 +1,12 @@
+VERSION = 0.1.1
```

Comment:
Down the road, the intention is to replace `lr-basic` with this image.
You'll see here that the only meaningful difference with `lr-basic` is it installs `google-cloud-cli` not `google-cloud-sdk`.